### ISSUE TITLE
In scan mode, NVDA does not announce app details on invoking 'focus without Elements' button.

### ISSUE THREAD

**Issue by msftedad** [nvaccess/nvda#15564]
### Steps to reproduce:

1. Open the URL- [NVDA iframe focus (codepen.io)](https://codepen.io/saenzzzup/pen/MWZGpep)
2. Activate NVDA 
3.  Select title of the page "iframe Focusable"
4. Tab to the first button (Focus Elements) and press enter
5. When Tab again it should start the announce of the title of the focus iframe.
6. Select title of the page "iframe Focusable"
7. Tab to the second button (Focus without Elements) and press enter.
8. The NVDA won't announce the tittle of the iframe and the next tab will jump out of the iframe.

### Actual behavior:
NVDA is not announcing the title of the iframe and the next tab will jump out of the iframe.

### Expected behavior:
In scan mode, NVDA should announce app details on invoking 'focus without Elements' button and on pressing tab key focus should land on controls..

### NVDA logs, crash dumps and other attachments:

https://github.com/nvaccess/nvda/assets/93735775/ca378004-77fd-48f6-8f91-d00744572ecc

#### NVDA version:
NVDA Version 2023.2(2023.2.0.29051)

#### Windows version:
Windows 11 Enterprise 22H2 Version: 22621.2283

#### Does the issue still occur after restarting your computer?
yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
NO
#### If NVDA add-ons are disabled, is your problem still occurring?
Yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

**Comment by seanbudd** [comment]
NVDA does not have a scan mode. Do you mean focus or browse mode?

**Comment by msftedad** [comment]
Hi @seanbudd, apologize for that. It is Focus mode(Tab key). Thanks

**Comment by seanbudd** [comment]
Can you please confirm what browser you are using and what browser version? We cannot reproduce this with Firefox

**Comment by msftedad** [comment]
Hi @jcsteh , please provide update on this issue. Thanks

**Comment by msftedad** [comment]
Hi @jcsteh , please provide update on this issue. Thanks

**Comment by Adriani90** [comment]
Relating to my https://github.com/nvaccess/nvda/issues/15564#issuecomment-1755692039
is there anything Firefox can do to improve this? Should we open new issues with Bugzilla or with CRbug tracker?

**Comment by seanbudd** [comment]
Hi @msftedad - can you please answer the questions requested in (https://github.com/nvaccess/nvda/issues/15564#issuecomment-1754037270)

>Can you please confirm what browser you are using and what browser version? We cannot reproduce this with Firefox
